pkgs/development/ocaml-modules/plotkicadsch/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, substituteAll
+, base64
+, cmdliner
+, digestif
+, git-unix
+, kicadsch
+, lwt
+, lwt_ppx
+, sha
+, tyxml
+, coreutils
+, imagemagick
+}:
+
+buildDunePackage rec {
+  pname = "plotkicadsch";
+
+  inherit (kicadsch) src version;
+
+  minimalOCamlVersion = "4.09";
+
+  patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
+      inherit coreutils imagemagick;
+    })
+  ];
+
+  buildInputs = [
+    base64
+    cmdliner
+    # digestif.c is what appears in the dune file
+    digestif
+    git-unix
+    kicadsch
+    lwt
+    lwt_ppx
+    sha
+    tyxml
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/jnavila/plotkicadsch";
+    description = "A small tool to export Kicad Sch files to SVG pictures";
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.gpl2Only;
```

https://github.com/jnavila/plotkicadsch/blob/master/LICENSE.md

Can you confirm the license?

---

https://github.com/jnavila/plotkicadsch/blob/master/plotkicadsch.opam#L13

---

I was looking at https://github.com/jnavila/plotkicadsch/blob/e5f94e479a620f9b86df82c023a1b9dec8e6a695/COPYING#L1-L29 when I wrote that. I really don't know which to choose or if we should simply use both.

---

`git blame` in the opam file you linked shows that upstream changed the license in that file from GPL2 to ISC, so that is what we'll go with.